ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.12__periodics.yaml

```diff
@@ -15,6 +15,10 @@ base_images:
     name: ocm-2.7
     namespace: edge-infrastructure
     tag: assisted-installer-controller
+  assisted-service:
```

Comment:
I think we can drop 4.12.
There's no need to test it

Reply:
done